Index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="Style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <nav class="navbar">
+    <h2 class="myLogo">My logo</h2>
+    <img class="burger" src="Assests/Union.jpg">
+  </nav>
+  <section class="sectionOne">
+    <h1 class="introTitle">I'm Kevin. Glad to see you!</h1>
+    <h3 class="textIntro">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</h3>
+    <h4 class="connect">LET'S CONNECT</h4>
+    <ul class="icons">
+      <li class="twitter"><img src="Assests/Vectortwitter.svg"></li>
+      <li><img src="Assests/Linkedin iconlinked.svg"></li>
+      <li class="medium"><img src="Assests/Vectormedium.svg"></li>
+      <li><img src="Assests/Vectorgithub.svg"></li>
+      <li><img src="Assests/Vectorangerllist.svg"></li>
+    </ul>
+  </section>
```

Comment:
- [ ] Overall, your design looks far different from the Figma design. The image below tries to point out the issues. They are mostly font-size issues and spacing between elements. Kindly compare the two images and see the difference, fix it. 

Your design | Figma Design
---- | ----
![image](https://user-images.githubusercontent.com/84156714/235035776-c4ba38dc-fff9-4cee-b3c3-61dbc209d38b.png) | ![image](https://user-images.githubusercontent.com/84156714/235035311-c54d1adc-a99a-447d-abae-bc6f9def63ab.png)